client/src/actions/newGrant.ts

```diff
@@ -67,12 +67,12 @@ export const publishGrant =
       if (grantId !== undefined) {
         projectTx = await projectRegistry.updateProjectMetaData(grantId, {
           protocol: 1,
-          pointer: state.ipfs.lastFileSavedCID,
+          pointer: state.ipfs.projectFileSavedCID,
```

Comment:
After a talk with @gravityblast he suggested moving this into the newGrant store. I created a ticket to do so here: https://github.com/gitcoinco/grants-hub/issues/97